src/In.ProjectEKA.HipService/Migrations/20200110085309_intial.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace hip_service.Migrations
+{
+    using Microsoft.EntityFrameworkCore.Migrations;
+
+    public partial class intial : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)